backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/image/TransferDiskImageCommand.java

```diff
@@ -193,6 +195,12 @@ protected boolean validateImageTransfer() {
                 validate(diskValidator.isDiskExists())
                 && validate(diskImagesValidator.diskImagesNotIllegal())
                 && validate(storageDomainValidator.isDomainExistAndActive());
+
+        if(diskImage.getDiskStorageType() == DiskStorageType.MANAGED_BLOCK_STORAGE
```

Comment:
doing this in validateImageTransfer will only work if the image is provided, which is always the case, but if support is somehow added to the UI it will not work, so perhaps it's good to also check just the storage domain in `validateCreateImage`